<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapWindow</class>
 <widget class="QMainWindow" name="MapWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MapWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="mapImg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>971</width>
      <height>531</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSearch">
     <property name="title">
      <string>Search</string>
     </property>
     <widget class="QMenu" name="menuAvailable_Beds">
      <property name="title">
       <string>Available Beds</string>
      </property>
      <addaction name="actionAcute_Care"/>
      <addaction name="separator"/>
      <addaction name="actionComplex_Continuing_Care"/>
      <addaction name="separator"/>
      <addaction name="actionLong_Term_Care"/>
     </widget>
     <addaction name="menuAvailable_Beds"/>
    </widget>
    <addaction name="menuSearch"/>
    <addaction name="separator"/>
    <addaction name="actionLogout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuFacility">
     <property name="title">
      <string>Facility</string>
     </property>
     <addaction name="actionHospital"/>
     <addaction name="separator"/>
     <addaction name="actionNursing_Home"/>
    </widget>
    <widget class="QMenu" name="menuZoom">
     <property name="title">
      <string>Zoom</string>
     </property>
     <addaction name="actionZoom_In"/>
     <addaction name="separator"/>
     <addaction name="actionZoom_Out"/>
    </widget>
    <addaction name="menuFacility"/>
    <addaction name="separator"/>
    <addaction name="actionFull_Screen"/>
    <addaction name="separator"/>
    <addaction name="menuZoom"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuAdmin">
    <property name="title">
     <string>Admin</string>
    </property>
    <addaction name="actionAdd_newUser"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_Facility"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuAdmin"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHospital">
   <property name="text">
    <string>Hospital</string>
   </property>
  </action>
  <action name="actionNursing_Home">
   <property name="text">
    <string>Nursing Home</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="text">
    <string>Full Screen</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionOn">
   <property name="text">
    <string>On</string>
   </property>
  </action>
  <action name="actionOff">
   <property name="text">
    <string>Off</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionAcute_Care">
   <property name="text">
    <string>Acute Care</string>
   </property>
  </action>
  <action name="actionComplex_Continuing_Care">
   <property name="text">
    <string>Complex Continuing Care</string>
   </property>
  </action>
  <action name="actionLong_Term_Care">
   <property name="text">
    <string>Long Term Care</string>
   </property>
  </action>
  <action name="actionAdd_newUser">
   <property name="text">
    <string>Create User</string>
   </property>
  </action>
  <action name="actionDelete_User">
   <property name="text">
    <string>Delete User</string>
   </property>
  </action>
  <action name="actionCreate_Facility">
   <property name="text">
    <string>Create Facility</string>
   </property>
  </action>
  <action name="actionDelete_Facility">
   <property name="text">
    <string>Delete Facility</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
